scripts/utils/verify_scripts.js

```diff
@@ -11,20 +11,40 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
   )
   const { getMasterCopy, getFallbackHandler } = require("./internals")(web3, artifacts)
   const { getDexagPrice, checkNoProfitableOffer } = require("./price_utils")(web3, artifacts)
+  const { equalUpToOrder } = require("./js_helpers")
 
   const GnosisSafe = artifacts.require("GnosisSafe.sol")
   const GnosisSafeProxy = artifacts.require("GnosisSafeProxy.sol")
   const gnosisSafeMasterCopy = GnosisSafe.deployed()
 
   const pageSize = 50
 
-  const verifyBracketsWellFormed = async function(masterAddress, bracketAddresses, logActivated = false) {
+  const verifyBracketsWellFormed = async function(
+    masterAddress,
+    bracketAddresses,
+    masterThreshold,
```

Comment:
Maybe you wanna indicate that these values do not have to be set by providing default values:
```
masterThreshold = null
```